docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
             # Devices API download endpoint
             - traefik.http.routers.deploymentsDL.entrypoints=https
             - traefik.http.routers.deploymentsDL.rule=PathPrefix(`/api/devices/{(v[0-9]+)}/deployments/download`)
-            - traefik.http.routers.deploymentsDL.middlewares=sec-headers,json-error-responder1,json-error-responder2,json-error-responder3,json-error-responder4
+            - traefik.http.routers.deploymentsDL.middlewares=sec-headers,compression,json-error-responder1,json-error-responder2,json-error-responder3,json-error-responder4
```

Comment:
This one breaks the configuration test, download the configuration artifact as "chunked".

Reply:
argh right, thanks! should now be removed...